examples/nativemodules/PureNativeModule.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { fireEvent, render } from 'react-native-testing-library';
+import PureNativeModule from './PureNativeModule';
+
+jest.mock('react-native', () => {
+  const RN = jest.requireActual('react-native');
+  RN.NativeModules.NativeNotification = {
+    show: jest.fn(),
+  };
+
+  return RN;
+});
+
+// Error thrown by import-order. Ignored due to the mock
+// eslint-disable-next-line
+import RN from 'react-native';
```

Comment:
`jest.mock` is hosited above imports (which are transformed to `require`s). Also, this import is not needed at all, please remove